src/xarray_regrid/methods/conservative.py

```diff
@@ -107,6 +113,7 @@ def conservative_regrid_dataarray(
     latitude_coord: str,
 ) -> xr.DataArray:
     """DataArray implementation of the conservative regridding method."""
+    attrs = data.attrs
```

Comment:
Need to extract coordinates' attributes.

```suggestion
    data_coords: list[str] = list(data.coords)
    datacoords = [data[var] for var in data_coords]
    attrs = data.attrs
    dcrds_attrs = [dcrds.attrs for dcrds in datacoords]
```